### PR TITLE
Copy to Clipboard

### PR DESCRIPTION
**Pull request by loverajoel**
## TL;DR;

This week I had to create a common "Copy to Clipboard" button, I've never created one before and I want to share with I made it.
## Username

@loverajoel
## Extra

np

### CODE REVIEW

**Comment by kurtextrem** [comment]
Can you add a caniuse.com link?

**Comment by loverajoel** [comment]
@kurtextrem done

File: _posts/en/2016-08-02-copy-to-clipboard.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+tip-number: 55
+tip-username: loverajoel
+tip-username-profile: https://twitter.com/loverajoel
+tip-tldr: This week I had to create a common "Copy to Clipboard" button, I've never created one before and I want to share with I made it.
+
+categories:
+    - en
+---
+
+This is a simple tip, this week I had to create a common "Copy to Clipboard" button, I've never created one before and I want to share with I made it.
```
Comment (kurtextrem) [inline _posts/en/2016-08-02-copy-to-clipboard.md]: "want to share with I made it" seems like a typo :)

File: _posts/en/2016-08-02-copy-to-clipboard.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+It's easy, the bad thing is that we must have to add an `<input/>` with the text to be copied to the DOM. Then, we selected the content and executes the copy command with [execCommand](https://developer.mozilla.org/en-US/docs/Web/API/Document/execCommand).
```
Comment (kurtextrem) [inline _posts/en/2016-08-02-copy-to-clipboard.md]: "must have to add" does not work in this context, better would be "we must add" or "we have to add"
"and executes" -> "and execute" (we execute, so no third person s)

**Comment by loverajoel** [comment]
@kurtextrem done! Thanks for your grammar corrections!

**Comment by fanboke** [comment]
谢谢分享

**Comment by hingsir** [comment]
don't work on iOS

**Comment by kurtextrem** [comment]
:shipit:

**Comment by zenopopovici** [comment]
:shipit:

**Comment by zenopopovici** [comment]
Maybe we should just :shipit: stuff and correct later. We have lots of backlog items.

@hingsir It does. Check caniuse.

**Comment by loverajoel** [comment]
Thanks!!!

**Comment by hingsir** [comment]
@loverajoel @zenopopovici I think safari on iOS supports `execCommand` but not `copy/cut` . Check [MDN](https://developer.mozilla.org/en-US/docs/Web/API/Document/execCommand) and test that on your devices.

**Comment by zenopopovici** [comment]
@hingsir Yep, it seems the clipboard cannot be read/written programatically. Feel free to submit a PR to amend the tip.

**Pull request merged**